constraints.txt

```diff
@@ -0,0 +1,82 @@
+aniso8601==3.0.2
```

Comment:
May we call this file `requirements-lock.txt`, in order to comply with other dependency management systems like `npm`, and also keeping it lexically close to the other `requirements` files

Reply:
I don't have strong opinion about this, @ulope is there a standard for this?

Reply:
I don't have a strong preference either (and don't know of a standard). Bu am leaning slightly towards keeping `constraints.txt` as that's the name used by pip itself:

```
-c, --constraint <file>     Constrain versions using the given constraints file. This option can be used multiple times.
```